src/gui/clips/ClipView.cpp

```diff
@@ -314,8 +314,7 @@ void ClipView::updateLength()
 	}
 	else
 	{
-		// 3 is the minimun width needed to paint a clip
```

Comment:
Technically the border width is 2 pixels so minimum width of a clip would be 5. But it works even if pixelsPerBar is 1. So I don't see a need to enforce a minimum here... Just don't let zoom get to small. Or am I missing something?

Reply:
That was needed for clips that do not start or end at beat times (Alt+mouse, to move start or end of the clip) and an "extreme" zoom in (clip disappears). It happens with your code as well. I think we should reintroduce that control again.